hack/update-codecgen.sh

```diff
@@ -33,6 +33,14 @@ generated_files=$(
       \) -prune \
     \) -name '*.generated.go')
 
+# Build codecgen binary from Godeps.
+function cleanup {
+  rm -rf "${KUBE_ROOT}/codecgen_binary"
+}
+trap cleanup EXIT
+go build -o codecgen_binary ../../github.com/ugorji/go/codec/codecgen
```

Comment:
What about pflag?  Seems like that will still come from the system GOPATH, right?